.github/workflows/push-test.yml

```
@@ -14,6 +14,22 @@ jobs:
         run: |
           python3 scripts/check_version_numbers.py
 
+  # Ensure correct Rust code formatting
+  fmt:
```

```suggestion
  formatting:
```
Just to be sure everybody understands what this action does :)